soccer_ipm/soccer_ipm/msgs/field_boundary.py

```diff
@@ -46,12 +47,16 @@ def map_field_boundary(
     points_np = np.array([[p.x, p.y] for p in msg.points])
 
     # Map all points at once from image onto field plane
-    points_on_plane = ipm.map_points(
-        field,
-        points_np,
-        msg.header.stamp,
-        plane_frame_id=output_frame,
-        output_frame_id=output_frame)[1]
+    try:
+        points_on_plane = ipm.map_points(
+            field,
+            points_np,
+            msg.header.stamp,
+            plane_frame_id=output_frame,
+            output_frame_id=output_frame)[1]
+    except CameraInfoNotSetException:
+        logger.warn('Inverse perspective mapping should be performed, \
+                but no camera info was recived yet!', throttle_duration_sec=5)
```

Comment:
I am not certain, how this gets shown by the logger, but breaking up a string like this in normal python leads to a lot of spaces before `but...`.
Rather, use `+` for concatenation.